wireless_modules/middle_module/mqtt_client.py

```diff
@@ -0,0 +1,72 @@
+
+# Use micro pip to install packages
+import upip
+upip.install("micropython-umqtt.simple2")
+from umqtt.simple import MQTTClient
```

Comment:
We still need `upip` when the board needs to download the micropython library for the first time

Reply:
That's strange. I was able to delete `import upip` and `upip.install(...)`, and delete all the downloaded libraries, and it still ran. Are you sure that `umqtt.simple` isn't installed by default?

Reply:
I did come across umqtt.robust2, and was planning to discuss if we should use it instead. Happy to look at it in this PR cos it directly affects the main purpose of this PR which is to get an MQTT Client class sorted. What do you think, worth going into in this PR?

Reply:
I reckon it's relevant so absolutely, go for it! I found I needed to use `import libs.umqtt.robust2`, that may or may not save you some time depending on if you've worked that out or not :joy:

Reply:
hahaha that probably did save a lot of my time, cheers. And in case if you have tried using robust2 already, did you have to still download umqtt.simple2, because apparently robust2 just builds on the simple2 class

Reply:
I have a feeling that it downloads `simple2` automatically. No harm in trying without doing it manually, and adding it in if it doesn't work!

Reply:
> That's strange. I was able to delete `import upip` and `upip.install(...)`, and delete all the downloaded libraries, and it still ran. Are you sure that `umqtt.simple` isn't installed by default?

How did you delete the downloaded libraries from the board? All my attempts don't seem to work